File: backend/src/main/java/br/com/unimedceara/portalcomunicacao/organization/domain/model/EquipeStatus.java
package br.com.unimedceara.portalcomunicacao.organization.domain.model;

/**
 * Status lógico da equipe organizacional.
 */
public enum EquipeStatus {
    ACTIVE,
    INACTIVE;

    public static EquipeStatus fromFlag(String flag) {
        return "S".equalsIgnoreCase(flag) ? ACTIVE : INACTIVE;
    }

    public String toFlag() {
        return this == ACTIVE ? "S" : "N";
    }
}

File: frontend/src/composables/organization/equipe-status.ts
import type { EquipeStatus } from "@/types/organization/equipe.types";

export function resolveTargetEquipeStatus(
  currentStatus: EquipeStatus
): EquipeStatus {
  return currentStatus === "ACTIVE" ? "INACTIVE" : "ACTIVE";
}

export function isEquipeDeactivation(currentStatus: EquipeStatus): boolean {
  return currentStatus === "ACTIVE";
}
